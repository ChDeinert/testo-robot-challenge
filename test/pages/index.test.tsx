import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Index from '../../src/pages';

describe('<Index Page/>', () => {
  it('renders correctly', () => {
    const { container } = render(<Index />);
    expect(container).toMatchSnapshot();
  });
});