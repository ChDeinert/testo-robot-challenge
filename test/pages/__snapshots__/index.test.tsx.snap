// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index Page/> renders correctly 1`] = `
<div>
  <main>
    <h1>
      Testo Robot Challenge
    </h1>
    <section
      class="board"
    >
      <div
        class="board__tile"
        data-coordinates="0 4"
        data-testid="boardTile04"
      />
      <div
        class="board__tile"
        data-coordinates="1 4"
        data-testid="boardTile14"
      />
      <div
        class="board__tile"
        data-coordinates="2 4"
        data-testid="boardTile24"
      />
      <div
        class="board__tile"
        data-coordinates="3 4"
        data-testid="boardTile34"
      />
      <div
        class="board__tile"
        data-coordinates="4 4"
        data-testid="boardTile44"
      />
      <div
        class="board__tile"
        data-coordinates="0 3"
        data-testid="boardTile03"
      />
      <div
        class="board__tile"
        data-coordinates="1 3"
        data-testid="boardTile13"
      />
      <div
        class="board__tile"
        data-coordinates="2 3"
        data-testid="boardTile23"
      />
      <div
        class="board__tile"
        data-coordinates="3 3"
        data-testid="boardTile33"
      />
      <div
        class="board__tile"
        data-coordinates="4 3"
        data-testid="boardTile43"
      />
      <div
        class="board__tile"
        data-coordinates="0 2"
        data-testid="boardTile02"
      />
      <div
        class="board__tile"
        data-coordinates="1 2"
        data-testid="boardTile12"
      />
      <div
        class="board__tile"
        data-coordinates="2 2"
        data-testid="boardTile22"
      />
      <div
        class="board__tile"
        data-coordinates="3 2"
        data-testid="boardTile32"
      />
      <div
        class="board__tile"
        data-coordinates="4 2"
        data-testid="boardTile42"
      />
      <div
        class="board__tile"
        data-coordinates="0 1"
        data-testid="boardTile01"
      />
      <div
        class="board__tile"
        data-coordinates="1 1"
        data-testid="boardTile11"
      />
      <div
        class="board__tile"
        data-coordinates="2 1"
        data-testid="boardTile21"
      />
      <div
        class="board__tile"
        data-coordinates="3 1"
        data-testid="boardTile31"
      />
      <div
        class="board__tile"
        data-coordinates="4 1"
        data-testid="boardTile41"
      />
      <div
        class="board__tile"
        data-coordinates="0 0"
        data-testid="boardTile00"
      />
      <div
        class="board__tile"
        data-coordinates="1 0"
        data-testid="boardTile10"
      />
      <div
        class="board__tile"
        data-coordinates="2 0"
        data-testid="boardTile20"
      />
      <div
        class="board__tile"
        data-coordinates="3 0"
        data-testid="boardTile30"
      />
      <div
        class="board__tile"
        data-coordinates="4 0"
        data-testid="boardTile40"
      />
    </section>
    <div
      class="error"
    />
    <p>
      loading...
    </p>
  </main>
</div>
`;
